package.json

```diff
@@ -10,9 +10,10 @@
     "node": ">=10"
   },
   "scripts": {
-    "build": "./lib/index.js",
+    "clean": "gulp clean",
+    "build": "gulp build",
+    "watch": "gulp watch & yarn start",
```

Comment:
```suggestion
     "dev": "gulp watch & npm start",
```

Reply:
Suggest changing this to "dev" to match what other CLI's like Next.js use for similar functionality.

Reply:
I don't think this exits cleanly. When you ctrl-c it kills the server, but gulp watch continues to run in the background.

Reply:
done